### DISCUSSION TITLE
Nestjs - page title as an option

### DISCUSSION THREAD

**Discussion by SnowMarble** [scalar/scalar#2180] [Ideas]
https://github.com/scalar/scalar/blob/40d7c3dd56dc15832d62c632be3c1afa8fdb3c02/packages/nestjs-api-reference/src/nestJSApiReference.ts#L111-L118

Currently it is not possible to change the document title in the Nestjs adapter. I think it's good to make it an option.

**Comment by hanspagel** [comment]
I think that should be possible with the upcoming release if you pass `metaData: { title: … }` to the configuration. See #1183

**Reply by SnowMarble** [reply]
yes. I found something's going on and close it, but I appreciate it.
